### ISSUE TITLE
All dependencies needed?

### ISSUE THREAD

**Issue by sahibalejandro** [sboo/multiauth#2]
I think Laravel install all dependencies listed on the composer.json, if it's the case, then they can be omitted on this package?

Here is the dependencies listed on the composer.json (at this time).

```
"require": {
      "php": ">=5.4.0",
        "illuminate/auth": "5.0.*",
        "illuminate/console": "5.0.*",
        "illuminate/database": "5.0.*",
        "illuminate/filesystem": "5.0.*",
        "illuminate/support": "5.0.*"
    }
```

**Comment by sboo** [comment]
having those dependencies there is necessary to
A) assure that you have Laravel installed and
B) the right version of Laravel (namely 5.0)

It doesn't mean that, when adding this package, it will install those same files again. 
Should you try to use the package in a different, non-Laravel environment, it would install the required dependencies.

**Comment by ldynia** [comment]
Hi @sboo I have tested the authentication It works flawlessly. Grate job! The brake trough for me came when I saw your gist.  replacing l5 Guard with Auth. Thanks and have a splendid day!

**Comment by ferm12** [comment]
@ldynia  Could you elaborate a little more how you made Laravel 5 to work with this page ("sboo/multiauth" : "4.0.*") . What do you mean in your response above when you say "...replacing I5 Guard with Auth..". Please, help I am having trouble making "sboo/multiauth" : "4.0.*" work with Laravel 5.0

**Comment by sboo** [comment]
@ferm12 Please read the **usage** section. There is even a link to a gist explaining exactly that.

**Comment by ldynia** [comment]
@ferm12 exacty as the @sboo said. Additionally, here is the gist with comments https://gist.github.com/ldynia/2abcd8a2c665361dd33a Cheers!

**Comment by ferm12** [comment]
@sboo good news, after struggling to make L5 to work with this project (sboo:multiauth:4.0), I finally figure out what was wrong with my installation. I follow word by word the USE section, and copied and pasted https://gist.github.com/sboo/10943f39429b001dd9d0, but I did not realize that I had to change every instance of Auth::admin() to whatever I have set up in my config/auth.php file, like @sboo suggested. In my case I used what is in the installation page . i.e, user . For people that are having the same problem as me, do what I did copy and paste what is in the gist, but remember Auth:admin() means that the Auth Class is going to look for a admin table and therefore a Admin Class and if you don't have it is going to throw an error.
